### ISSUE TITLE
[2.1.6] iPhone 6S / 10.2 - Uploading is getting failed when user logout from the app while uploading.

### ISSUE THREAD

**Issue by Pankaj-Testershub** [Lomotif/lomotif-ios-issues#1323]
Frequency: Every time

Action:
1; Launch the app an log in with valid credential.
2: Tap on "START" and create lomotif with video/motifs.
3: Proceed to "Create" screen with/without music and tap on export option.
4: Skip sharing and navigate to "Lomotifs" tab(Uploading progres will be displayed).
5: Log out from the app and navigate to "Profile" screen("In progress" tab).
6: Wait for few secs.
7: Check if uploading is getting failed.

Expected result: Uploading should not get fail when user logout from the app while uploading.

Actual result: Uploading is getting failed when user logout from the app while uploading.

Refer following video link for more details:
https://drive.google.com/open?id=0B0-oVt0Gfy3ib2dQN2lYc2dtZHM

**Comment by testershub** [comment]
@xiuie I can reproduce this issue for build [2.1.6] on iPhone 5S / 9.3.5.

**Comment by hiteshbheda** [comment]
@xiuie I think it's working as expected and not a bug. Because when user log out from lomotif app account then Uploading process for that specific user should be stopped.

I can reproduce this issue for build [2.1.6] on iPhone 6S / iOS 10.2

**Comment by xiuie** [comment]
In future, we will have a dialogue box warning them that their upload progress will be interrupted.

**Comment by hiteshbheda** [comment]
@xiuie Yes, I think it would be better to display popup regarding interrupted service.
